src/docs/sphinx/conf.py.in

```diff
@@ -52,9 +52,9 @@ copyright = u'2019-2021, LLNS'
 # built documents.
 #
 # The short X.Y version.
-version = '0.4.1'
+version = '0.5.0'
```

Comment:
We have traditionally updated the version in a commit where we are bumping the release version throughout the entire project. I have already captured an update to the version here: https://github.com/LLNL/variorum/pull/251